packages/theme/css/characteristics/overlayable.css

```diff
@@ -1,6 +1,6 @@
 .salt-theme {
   --salt-overlayable-shadow-scroll: var(--salt-shadow-1);
-  --salt-overlayable-shadow-borderRegion: var(--salt-shadow-2);
+  --salt-overlayable-shadow-region: var(--salt-shadow-2);
   --salt-overlayable-shadow: var(--salt-shadow-2);
```

Comment:
whats the difference between ```--salt-overlayable-shadow-region``` and ```--salt-overlayable-shadow``` ? If the same then i would prefer ```--salt-overlayable-shadow```

Reply:
good spot - let me check with design if this was intended

Reply:
they have different semantics in the characteristic, one is default, one for regions. when customising theme, people can point the region or default to a different foundation value if they differ in their theme

Reply:
ah nice thanks @libertymayc good to know. Paul is aware of the 2 tokens pointing to the same values, just waiting on him to get back on design decisions

Reply:
Confirmed with design - the 2 tokens are distinct as they're intended to be semantic. 
Quoting Ben,
_So from a theming perspective, teams can choose to alter the values independently, if they choose. Thinking being that a 'region' doesn't necessarily have to be the same as a 'card'. We only have so many shadows in our foundational ramp, so for Salt they use the same value --shadow-200, but a consumer might choose to set this themselves._